### DISCUSSION TITLE
Node tree looks different from everybody else.

### DISCUSSION THREAD

**Discussion by AlexArisu** [bnpr/Malt#163] [Q&A] [answered]
I've been looking all around and my nodes looks different than others. Could it be I downloaded the wrong version? I've tried the release version and dev version. I have no coding/scripting background and I really dont wanna waste your precious time, is there a place u can point me to to fix this problem?

I notice the PixelOutput.id is in uvec4 instead of float 
![blender_GiYjvFNUj4](https://user-images.githubusercontent.com/94103097/141259474-52fcc52d-b77f-499a-8fe6-9ee1c2004c05.png)

I tried to follow the malt nodes introduction but the get_diffuse_half makes the color turns black
![blender_yqYwBPXjwi](https://user-images.githubusercontent.com/94103097/141260375-f3c83568-a40d-4a01-9dbf-b75b881fe1a0.png)
![blender_D6tqKbIrl4](https://user-images.githubusercontent.com/94103097/141260389-4dc566f7-e3b4-493e-a9e5-82b7bbe77980.png)
. Thanks for your amazing work, I really hope I can use it well.

**Comment by pragma37** [comment, marked as answer]
> I've been looking all around and my nodes looks different than others. Could it be I downloaded the wrong version?

I'm currently working on improving the node UX (#148), so the Development branch has some differences/incompatibilities with previous versions.
Here you can see a comparison:
https://twitter.com/pragma37/status/1453420042654257157

If you are getting started with Malt you may have an easier time with the Release branch.
https://github.com/bnpr/Malt/releases/tag/Release-latest
You can check the sample files as well: #94 

> I tried to follow the malt nodes introduction but the get_diffuse_half makes the color turns black

My bet is you are using a point light that is too far from the model. The default point light radius in Malt is 5 meters.

**Reply by AlexArisu** [reply]
Thank you for your reply, I've tried out your advice and everything worked out as you explained! Thanks a lot!
